modules/dnn/perf/perf_einsum.cpp

```diff
@@ -0,0 +1,121 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "perf_precomp.hpp"
+
+namespace opencv_test {
+
+struct EinsumParams {
+    int inputSize;
+    int outputSize;
+    std::string equation;
+    std::vector<MatShape> einsumInpShapes;
+    EinsumParams(std::string equation_, int inputSize_, int outputSize_,  std::vector<MatShape> einsumInpShapes_ = std::vector<MatShape>())
+    {
+        inputSize = inputSize_;
+        outputSize = outputSize_;
+        equation = equation_;
+        einsumInpShapes = einsumInpShapes_;
+    }
+};
+
+static inline void PrintTo(const EinsumParams& params, ::std::ostream* os) {
+     (*os) << "\nEqiation: \t" << params.equation << "\n"
+        << "InputSize: \t" << params.inputSize << "\n"
+        << "OutputSize: \t" << params.outputSize << "\n";
+
+        for(int i = 0; i < params.einsumInpShapes.size(); i++)
+        {
+            (*os) << "InputShape " << i << ": \t";
```

Comment:
The same here. I propose alternative format like `InputShape={{1, 4,70, 90}, {1, 5, 90, 80}}`